src/bundle.c

```diff
@@ -1058,13 +1058,190 @@ static struct list *get_dir_files_sorted(char *path)
 	return list_sort(files, lex_sort);
 }
 
+struct alias_lookup {
+	char *alias;
+	struct list *bundles;
+};
+
+/* Deep free on alias_lookup structs
+ */
+static void free_alias_lookup(void *lookup)
+{
+	struct alias_lookup *l = (struct alias_lookup *)lookup;
+	free(l->alias);
+	list_free_list_and_data(l->bundles, free);
+	free(l);
+}
+
+/* Get bundles for a given alias but if the alias is
+ * not actually an alias, return a list with just the
+ * bundle name
+ */
+static struct list *get_alias_bundles(struct list *alias_definitions, char *alias)
+{
+	struct list *bundles = NULL;
+	struct list *iter = alias_definitions;
+
+	while (iter) {
+		struct alias_lookup *lookup = (struct alias_lookup *)iter->data;
+		iter = iter->next;
+		if (strcmp(lookup->alias, alias) == 0) {
+			bundles = list_deep_clone_strs(lookup->bundles);
+			break;
+		}
+	}
+	if (!bundles) {
+		bundles = list_prepend_data(bundles, strdup_or_die(alias));
+	}
+
+	return bundles;
+}
+
+/* Parse an alias definition file for a list of bundles
+ * it specifies
+ *
+ * Example file:
+ * alias1	b1	b2
+ * alias2	b3	b4
+ */
+static struct list *parse_alias_file(char *fullpath)
```

Comment:
Okay. I'll move that code on, what did you think about the helper.c move that @castulo suggested?

Reply:
The perfect place for this file is sys.c, where we are grouping all system/filesystem related functions. But this didn't get merged yet, so leave it here and we can move to sys.c later.

Reply:
So I moved the get_dir_files_sorted to helpers.c for now because I felt it was easier to work with since the function is used in the alias.c file now. I'll leave it for folks to move to sys.c when things are in better shape if that is okay?

Reply:
yes. it's totally fine.